packages/jest-serializer/src/index.ts

```diff
@@ -109,55 +93,61 @@ function jsonStringify(content) {
   /* eslint-disable no-extend-native */
 
   try {
-    // $FlowFixMe: intentional removal of "toJSON" property.
+    // @ts-ignore intentional removal of "toJSON" property.
     Date.prototype.toJSON = undefined;
-    // $FlowFixMe: intentional removal of "toJSON" property.
+    // @ts-ignore intentional removal of "toJSON" property.
     Buffer.prototype.toJSON = undefined;
 
     return JSON.stringify(content, replacer);
   } finally {
-    // $FlowFixMe: intentional assignment of "toJSON" property.
     Date.prototype.toJSON = dateToJSON;
```

Comment:
Interesting that TS allows to rewrite Date prototype like this

Reply:
The function declaration is the same, so it's allowed.

Reply:
I thought it's non-configurable, but didn't check the definition tbh